charts/ocis/templates/web/deployment.yaml

```diff
@@ -74,6 +74,19 @@ spec:
             - name: WEB_UI_CONFIG_SERVER
               value: "https://{{ .Values.externalDomain }}"
 
+            - name: WEB_OPTION_CONTEXTHELPERS_READ_MORE
+              value: {{ .Values.services.web.config.contextHelpersReadMore.enabled | quote }}
+
+            {{- if .Values.features.externalUserManagement.oidc.sessionManagementLink }}
+            - name: WEB_OPTION_LOGOUT_URL
+              value: {{ .Values.features.externalUserManagement.oidc.sessionManagementLink | quote }}
+            {{- end }}
+
+            {{- if .Values.features.externalUserManagement.oidc.editAccountLink }}
+            - name: WEB_OPTION_ACCOUNT_EDIT_LINK_HREF
```

Comment:
`WEB_OPTION_ACCOUNT_EDIT_LINK_HREF ` is missing in https://doc.owncloud.com/ocis/next/deployment/services/s-list/web.html. This might not be an issue for this PR if it's really implemented and just missing in the documentation.